src/main/java/frc/robot/components/TrigonXboxController.java

```diff
@@ -3,13 +3,13 @@
 import edu.wpi.first.wpilibj.Notifier;
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import frc.robot.constants.RobotConstants;
 
 public class TrigonXboxController extends XboxController {
-    private static final double kIntermittentRumbleTime = 0.15;
-    private final JoystickButton buttonA;
-    private final JoystickButton buttonB;
-    private final JoystickButton buttonX;
-    private final JoystickButton buttonY;
+    private final JoystickButton a;
+    private final JoystickButton b;
+    private final JoystickButton x;
+    private final JoystickButton y;
```

Comment:
i feel like the names with button is better
you don't call topMotor: top

Reply:
I think this is debatable.

Reply:
and I say it's better as buttonA

Reply:
aBtn if you want to add it to the name